docs/Getting-Started-Guides/On-Premises-FPGA-Getting-Started/Repo-Setup/Xilinx-XDMA-Template.rst

```diff
@@ -0,0 +1,132 @@
+FireSim Repo Setup
+==============================
+
+.. |manager_machine| replace:: **Manager Machine**
+.. |build_farm_machine| replace:: **Build Farm Machines**
+.. |run_farm_machine| replace:: **Run Farm Machines**
+
+.. |mach_or_inst| replace:: Machine
+.. |mach_or_inst_l| replace:: machines
+.. |mach_details| replace:: your local desktop or server
+.. |mach_or_inst2| replace:: local machines
+.. |simple_setup| replace:: In the simplest setup, a single host machine (e.g. your desktop) can serve the function of all three of these: as the manager machine, the build farm machine (assuming Vivado is installed), and the run farm machine (assuming an FPGA is attached).
+
+
+In this step, we will perform final setup by cloning FireSim on your Manager Machine
+and completing setup in the repo.
+
+
+Setting up the FireSim Repo
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Machine:** From this point forward, run everything on your Manager Machine, unless otherwise instructed.
+
+We're finally ready to fetch FireSim's sources. This should be done on your Manager Machine. Run:
+
+.. code-block:: bash
+   :substitutions:
+
+    git clone https://github.com/firesim/firesim
+    cd firesim
+    # checkout latest official firesim release
+    # note: this may not be the latest release if the documentation version != "stable"
+    git checkout |overall_version|
+
+Next, we will bootstrap the machine by installing Miniforge Conda, our software package manager, and set up a default software environment using Conda.
+
+You should select a location where you want conda to be installed. This can be an existing Miniforge Conda
+install, or a directory (that does not exist) where you would like conda to be installed.
+
+Replace ``REPLACE_ME_USER_CONDA_LOCATION`` in the command below with your chosen path and run it:
+
+.. code-block:: bash
+
+   ./scripts/machine-launch-script.sh --prefix REPLACE_ME_USER_CONDA_LOCATION
+
+
+Among other setup steps, the script will install Miniforge Conda (https://github.com/conda-forge/miniforge) and create a default environment called ``firesim``.
+
+When prompted, you should allow the Conda installer to modify your ``~/.bashrc`` to automatically place you in the conda environment when opening a new shell.
+
+**Once the** ``machine-launch-script.sh`` **completes, ensure that you log out of the machine / exit out of the terminal so that the** ``.bashrc`` **modifications can apply**.
```

Comment:
```suggestion
.. warning::
    **Once the** ``machine-launch-script.sh`` **completes, ensure that you log out of the 
    machine / exit out of the terminal so that the** ``.bashrc`` **modifications can apply**.
```